frontend/lib/src/components/elements/ImageList/styled-components.ts

```diff
@@ -39,9 +39,6 @@ export const StyledImageContainer = styled.div(({ theme }) => ({
 }))
 
 export const StyledCaption = styled.div(({ theme }) => ({
-  fontFamily: theme.genericFonts.bodyFont,
```

Comment:
We can remove these styling properties because they are handled by `StreamlitMarkdown` if `isCaption` is set.